src/controllers/softtakeover.h

```diff
@@ -13,13 +13,11 @@
 
 class ControlObject;
 
-// 3/128 units away from the current is enough to catch fast non-sequential moves
-//  but not cause an audibly noticeable jump.
-
-
 class SoftTakeover {
   public:
-    static const double kDefaultTakeoverThreshold;
+    // 3/128 units away from the current is enough to catch fast non-sequential moves
+    //  but not cause an audibly noticeable jump.
+    static const double kDefaultTakeoverThreshold = 3.0 / 128;
```

Comment:
Sorry for the false positive. I have just tested it on Ubuntu Trusty and surprisingly to me it compiles. 
The same on Travis.